extract_unique_sensors_01.py

```diff
@@ -62,19 +62,19 @@ def extract_all_sensors(sensors_data, sensor_name):
     return candidate_lines
 
 # Loop over candidate_lines and extract sensor names
-def splitting_list(sensors_list):
+def splitting_list(sensor_data = read_sensors(file_name, url)):
     """Splits the candidate_lines by dot and append that to sensors_list. It also removes the 
     empty lines.
     """
     sensors_list = []
     candidate_lines = sensors_list
-    sensors_list = extract_all_sensors(sensors_data)
-    unique_sensors = splitting_list(sensors_list)
+    sensors_list = extract_all_sensors(sensor_data, sensor_name=kat_sensor)
```

Comment:
Do not call functions that are outside.
What would happen, if you choose to rename `extract_all_sensors` function?

**Do not call outside functions inside another function.**

Reply:
It means  that I will have to rename here as well. So should I parse it as argument?